concourse/pipelines/guest-package-build.jsonnet

```diff
@@ -463,6 +472,20 @@ local build_guest_configs = buildpackagejob {
             machine_type: 't2a-standard-2',
             worker_image: 'projects/compute-image-tools/global/images/family/debian-12-worker-arm64',
           },
+          buildpackageimagetask {
+            image_name: 'rhel-10',
+            source_image: 'projects/rhel-cloud/global/images/family/rhel-10',
```

Comment:
This will fail until we have a public rhel10 image.  You may want RHEL9 until it's working